### PR TITLE
leftover changes from PR 746

### PR DESCRIPTION
**Pull request by nsinkov**
## Changes proposed in this PR

- some small fixes 

## Why are we making these changes?

already committed to making these changes in PR 746

### CODE REVIEW

File: waiter/integration/waiter/authentication_test.clj [hunk 1]
```diff
@@ -27,8 +29,9 @@
 (deftest ^:parallel ^:integration-fast test-token-authentication-parameter-error
   (testing-using-waiter-url
     (when (using-composite-authenticator? waiter-url)
-      (let [authentication-providers (map name (keys (get-in (waiter-settings waiter-url)
-                                                             [:authenticator-config :composite :authentication-providers])))
+      (let [authentication-providers (map name (keys (get-in
```
Comment (shamsimam) [inline waiter/integration/waiter/authentication_test.clj]: thrading macro `->>` would make this easier to read

File: waiter/src/waiter/util/client_tools.clj [hunk 2]
```diff
@@ -987,6 +987,18 @@
   [waiter-url]
   (= "composite" (get-authenticator-kind waiter-url)))
 
+(defn using-saml-authenticator?
```
Comment (shamsimam) [inline waiter/src/waiter/util/client_tools.clj]: Should we make this private so someone doesn't inadvertently use this function?

Comment (nsinkov) [reply waiter/src/waiter/util/client_tools.clj]: if someone wants to use it, they can

File: waiter/integration/waiter/authentication_test.clj (hunk 1 above)
Comment (nsinkov) [reply waiter/integration/waiter/authentication_test.clj]: updated

File: waiter/src/waiter/util/client_tools.clj (hunk 2 above)
Comment (nsinkov) [reply waiter/src/waiter/util/client_tools.clj]: the function does what it says

**Comment by nsinkov** [comment]
@shamsimam  all green, can you merge

**Pull request merged**